### ISSUE TITLE
data-__neos-insertion-anchor throws an error expecting a closing tag 

### ISSUE THREAD

**Issue by 6180design** [sjsone/vscode-neos-fusion-lsp#10]
The underscores seems to be the problem here...

https://github.com/sjsone/vscode-neos-fusion-lsp/blob/a5a34a8304424c63a249d6b23e517f51a07eef09/server/src/diagnostics/DiagnoseTagNames.ts#L22

Example:
```
        <div data-__neos-insertion-anchor={props.addBackendMetadata}>
            {props.content}
        </div>
```

**Comment by sjsone** [comment]
Yes, that was an oversight on my part. I have tested and patched it in the versions **Release 0.3.16** and **Pre Release 0.5.3**. 

Can you confirm that everything is working now?

**Comment by 6180design** [comment]
That's perfect! Thanks for the quick fix!!
